specification/security/resource-manager/Microsoft.Security/preview/2017-08-01-preview/security.json

```diff
@@ -1235,6 +1369,32 @@
             "type": "string",
             "description": "name of the Compliance",
             "x-ms-parameter-location": "method"
+        },
+        "ResourceUri": {

```

Comment:
change to "resourceId" to use the same standard as other RPs

Reply:
thanks